capabilities/empowering-teams-to-choose-tools.md

```diff
@@ -1,51 +1,78 @@
+It’s crucial to balance tool choice with the potential costs of acquisition, support, and added complexity that come with adding new tools to a tech stack. 
 
-But if a team feels strongly that a different tool or technology is best for their case, they are free to choose it. They should know that their choice means they must also do the work of supporting this new tool or set of tools.
+Typically, organizations have platform teams that do a lot of the "behind the scenes" support work required to keep a technology stack compliant and efficient. A baseline set of approved tools should be established for all teams, which can cover the majority of the organization's needs while allowing room for innovation. If one team feels strongly that a different tool or technology is best for their case, they should feel free to choose it. However, they should know that their choice means they must also do the work of _supporting_ this new tool or set of tools.
 
-Sometimes that support work can be easy to gloss over. Take for example the decision to use a different language. The teams will need to build out a whole new CI/CD pipeline and ensure they have a environments suitable to run the code in the new language. They'll also need to introduce tooling that scans for security vulnerabilities and a host of other tasks that a company's platform team likely already handles.
-
-### Risk of Tool Overproliferation
+Sometimes that support work can be easy to gloss over. Take, for example, the decision to use a different language. Once this decision is made, the team will need to build out a whole new CI/CD pipeline and ensure its environments are suitable to run their code in the new language. They'll also need to introduce other new tooling that scans for security vulnerabilities and a host of other tasks, which the organization's platform team likely already handles. As you can see, one team's decision to use another tool has created a domino effect in terms of support, and has potentially added a layer of complexity in communication between teams. 
 
 **Allowing teams to choose their tools doesn't mean unrestricted freedom.**
 
-Too much freedom in tool choice can lead to a fragmented tech stack, increasing technical debt and fragility over time. Each new tool requires maintenance and integration efforts, which can dilute the benefits of the new technology. It's essential to define a process for exceptions and ensure teams understand the implications of their choices.
+Too much freedom in tool choice can lead to a fragmented tech stack, increasing technical debt and fragility over time. Each new tool requires maintenance and integration efforts, which can dilute the benefits of the new technology. It's essential to define a standardized process for evaluating and adopting new tools, and to ensure teams understand the implications of their choices.
+
+
+##Assessment
```

Comment:
For the markdown section styling to apply, there needs to be a space between the ## and the text that follows.

Reply:
Whoops! Addressed in 12/19 commit.